### PR TITLE
[ZH] Update `People` and nested articles

### PR DESCRIPTION
**Pull request by YumeMuzi**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
- [x] #10692 Merged

### CODE REVIEW

File: wiki/People/Beatmap_Nominators/Beatmap_Nominator_Test/zh.md [hunk 1]
```diff
@@ -27,7 +22,7 @@ outdated_since: 119117ef2cb9b730639df53a6ca5d3b8b9367c1a
 - [故事板标准](/wiki/Ranking_criteria#故事板)
 - 游戏模式特殊标准：与将申请的游戏模式相关 (osu!, osu!taiko, osu!catch, osu!mania)
 
-BN 考试没有分数线，但可以大概地根据得分把申请者分为三类：分数高于 15 为良好，分数处于 12.5 到 15 分之间为一般，低于 12.5 分为较差。
+对于 osu! 模式上的考试，如果分数低于 12.5，则算作未通过考试。对于其他三个游戏模式来说，BN 考试没有分数线，但可以大概地根据得分把申请者分为三类：分数高于 15 为良好，分数处于 12.5 到 15 分之间为一般，低于 12.5 分为较差。
```
Comment (CloneWith) [inline wiki/People/Beatmap_Nominators/Beatmap_Nominator_Test/zh.md:25]:
```suggestion
对于 osu! 模式的考试，如果分数低于 12.5，则算作未通过考试。对于其他三个游戏模式来说，BN 考试没有分数线，但可以根据得分大致把申请者分为三类：分数高于 15 为良好，分数处于 12.5 到 15 分之间为一般，低于 12.5 分为较差。
```
语序微调

File: wiki/People/Community_Contributors/zh.md [hunk 2]
```diff
@@ -227,6 +222,35 @@ outdated_translation: true
...
+
+#### 十一月
+
+*参见新闻报道：[Community Contributors: 2022 & 2023](https://osu.ppy.sh/home/news/2023-11-19-community-contributors-2022-2023)*
+
+| 玩家 | 贡献 |
+| :-- | :-- |
+| ::{ flag=AU }:: [Ephemeral](https://osu.ppy.sh/users/102335) | 在社区管理中，做出了十多年无休止的卓越贡献 |
+| ::{ flag=PL }:: [Venix](https://osu.ppy.sh/users/5999631) | 在摸图领域和[谱面聚光灯](/wiki/Beatmap_Spotlights)中做出了卓越贡献 |
+| ::{ flag=CH }:: [TicClick](https://osu.ppy.sh/users/672931) | 在摸图事务和 osu! wiki (wikifriend) 中做出了卓越贡献 |
```
Comment (CloneWith) [inline wiki/People/Community_Contributors/zh.md:235]: `wikifriend` 似乎指的是他在 osu! wiki 方面的称号，需要核实。

File: wiki/People/Nomination_Assessment_Team/zh.md [hunk 3]
```diff
@@ -1,18 +1,16 @@
...
 
 # 审核评估团队 (NAT)
 
-*参见：[BN 评估](Evaluations)，[SEV 分数](SEV_rating)*
+*参见：[BN 评估](Evaluations)*
 
 **审核评估团队** 全称为 Nomination Assessment Team (***NAT***)，是负责监督[谱面审核成员](/wiki/People/Beatmap_Nominators) (*BN*)，并确保 osu! 的谱面相关功能或社区保持正常的一群玩家。
 
-可以通过橙红色的游戏内名称、橙红色的玩家头衔和写着 `NAT` 的[用户组徽章](/wiki/People/User_group)认出他们。和[全局管理团队](/wiki/People/Global_Moderation_Team) (*GMT*) 一样，他们拥有全网站的管理和审核权限。同时，他们也和 [正式](/wiki/People/Beatmap_Nominators#正式成员)[谱面审核成员](/wiki/People/Beatmap_Nominators) (*BN*) 一样，能够提名谱面，或重置谱面的提名。大多数情况中，审核评估团队 (NAT) 成员就是具有不同职责的谱面审核成员 (*BN*)。因此，对于审核评估团队的所有成员，所有加在谱面审核成员 (*BN*) 身上的 [BN 规则](/wiki/People/Beatmap_Nominators/Rules)和[对 BN 的期望](/wiki/People/Beatmap_Nominators/Expectations)都适用。但是他们无需遵守类似活跃度要求的规则。
+可以通过橙红色的游戏内名称、橙红色的玩家头衔和写着 `NAT`，橙红色的[用户组徽章](/wiki/People/User_group)认出他们。和[全局管理团队](/wiki/People/Global_Moderation_Team) (*GMT*) 一样，他们拥有全网站的管理和审核权限。同时，他们也和 [正式](/wiki/People/Beatmap_Nominators#正式成员)[谱面审核成员](/wiki/People/Beatmap_Nominators) (*BN*) 一样，能够提名谱面，或重置谱面的提名。大多数情况中，审核评估团队 (NAT) 成员就是具有不同职责的谱面审核成员 (*BN*)。因此，对于审核评估团队的所有成员，所有加在谱面审核成员 (*BN*) 身上的 [BN 规则](/wiki/People/Beatmap_Nominators/Rules)和[对 BN 的期望](/wiki/People/Beatmap_Nominators/Expectations)都适用。但是他们无需遵守类似活跃度要求的规则。
```
Comment (CloneWith) [inline wiki/People/Nomination_Assessment_Team/zh.md:13]: 这里改错了，新版本没有后加的橙红色

File: wiki/People/Users_with_unique_titles/zh.md [hunk 4]
```diff
@@ -75,7 +70,7 @@ outdated_since: 09238489fa03b6decf831f6781f0cddff25b853d
...
 | [OWC 2020](/wiki/Tournaments/OWC/2020) | Former osu! Champion: 2020 | ::{ flag=US }:: [Apraxia](https://osu.ppy.sh/users/4194445), ::{ flag=US }:: [BTMC](https://osu.ppy.sh/users/3171691), ::{ flag=US }:: [kablaze](https://osu.ppy.sh/users/3043603), ::{ flag=US }:: [Rektygon](https://osu.ppy.sh/users/7813296), ::{ flag=US }:: [Vaxei](https://osu.ppy.sh/users/4787150) |
-| [OWC 2021](/wiki/Tournaments/OWC/2020) | Former osu! Champion: 2021 | ::{ flag=US }:: [Aireu](https://osu.ppy.sh/users/1650010), ::{ flag=US }:: [Aricin](https://osu.ppy.sh/users/1419095), ::{ flag=US }:: [DigitalHypno](https://osu.ppy.sh/users/4384207), ::{ flag=US }:: [Exarch](https://osu.ppy.sh/users/5370153), ::{ flag=US }:: [fieryrage](https://osu.ppy.sh/users/3533958) |
+| [OWC 2021](/wiki/Tournaments/OWC/2021) | Former osu! Champion: 2021 | ::{ flag=US }:: [Aireu](https://osu.ppy.sh/users/1650010), ::{ flag=US }:: [Aricin](https://osu.ppy.sh/users/1419095), ::{ flag=US }:: [DigitalHypno](https://osu.ppy.sh/users/4384207), ::{ flag=US }:: [Exarch](https://osu.ppy.sh/users/5370153), ::{ flag=US }:: [fieryrage](https://osu.ppy.sh/users/3533958) |
```
Comment (CloneWith) [inline wiki/People/Users_with_unique_titles/zh.md:73]: 下方漏了一行 2022 年的
`| [OWC 2022](/wiki/Tournaments/OWC/2022) | Former osu! Champion: 2022 | ::{ flag=US }:: [decaten](https://osu.ppy.sh/users/5645231), ::{ flag=US }:: [im a fancy lad](https://osu.ppy.sh/users/4908650), ::{ flag=US }:: [Monko2k](https://osu.ppy.sh/users/4852013), ::{ flag=US }:: [Sawada](https://osu.ppy.sh/users/8198848), ::{ flag=US }:: [Utami](https://osu.ppy.sh/users/7512553) |`

File: wiki/People/Beatmap_Nominators/zh.md [hunk 5]
```diff
@@ -61,196 +63,181 @@ BN 可获得以下特权：
...
 | ::{ flag=CA }:: [Sparhten](https://osu.ppy.sh/users/7601720) |  |
-| ::{ flag=BE }:: [Stack](https://osu.ppy.sh/users/6122935) | 荷兰语 |
-| ::{ flag=CA }:: [Terakomari](https://osu.ppy.sh/users/5312547) |  |
+| ::{ flag=CA }:: [superstore](https://osu.ppy.sh/users/16755693) |  |
 | ::{ flag=TH }:: [timemon](https://osu.ppy.sh/users/2072005) | 泰语 |
 | ::{ flag=LT }:: [tomatas95](https://osu.ppy.sh/users/11403815) | 立陶宛语 |
-| ::{ flag=US }:: [Usaha](https://osu.ppy.sh/users/6443117) |  |
+| ::{ flag=US }:: [Usaha](https://osu.ppy.sh/users/6443117) | 日语 |
 | ::{ flag=CA }:: [Vararaup](https://osu.ppy.sh/users/8257675) |  |
 | ::{ flag=US }:: [wafer](https://osu.ppy.sh/users/9416836) |  |
```
Comment (CloneWith) [inline wiki/People/Beatmap_Nominators/zh.md:110]:
```suggestion
| ::{ flag=US }:: [wafer](https://osu.ppy.sh/users/9416836) | 俄语 |
```

File: wiki/People/Beatmap_Nominators/zh.md [hunk 6]
```diff
@@ -61,196 +63,181 @@ BN 可获得以下特权：
...
+| ::{ flag=GB }:: [050](https://osu.ppy.sh/users/16352736) |  |
 | ::{ flag=HK }:: [autofanboy](https://osu.ppy.sh/users/636114) | 粤语 |
 | ::{ flag=CL }:: [Bastian](https://osu.ppy.sh/users/6345176) | 西班牙语 |
-| ::{ flag=HK }:: [BlackBN](https://osu.ppy.sh/users/6291741) | 汉语，粤语 |
+| ::{ flag=HK }:: [BlackBN](https://osu.ppy.sh/users/6291741) | 粤语，汉语 |
+| ::{ flag=NL }:: [Chatie](https://osu.ppy.sh/users/6524765) | 荷兰语 |
+| ::{ flag=AT }:: [Daletto](https://osu.ppy.sh/users/7592136) | 德语 |
 | ::{ flag=ID }:: [Dapulezatos](https://osu.ppy.sh/users/8140944) | 印度尼西亚语 |
+| ::{ flag=DE }:: [Du5t](https://osu.ppy.sh/users/6053071) |  |
 | ::{ flag=CA }:: [fuhie](https://osu.ppy.sh/users/7620002) |  |
```
Comment (CloneWith) [inline wiki/People/Beatmap_Nominators/zh.md:150]:
```suggestion
| ::{ flag=CA }:: [fuhie](https://osu.ppy.sh/users/7620002) | 德语 |
```

**Review by CloneWith — COMMENTED** [review]
That's quite a lot of work! thx ❤️‍🔥

File: wiki/People/Community_Contributors/zh.md (hunk 2 above)
Comment (YumeMuzi) [reply wiki/People/Community_Contributors/zh.md:235]: 正是因为这里看起来是个独特的称号，所以应该不需要翻译。

也可以直接问本人（反正他是 maintainer）

先 keep

File: wiki/People/Users_with_unique_titles/zh.md (hunk 4 above)
Comment (YumeMuzi) [reply wiki/People/Users_with_unique_titles/zh.md:73]: fixed

**Review by CloneWith — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**